README.md

```diff
@@ -1 +1,76 @@
-# papelillo
\ No newline at end of file
+# Papelillo
+
+Papelillo is a library to wrap and manage mailing lists, so you can create/update/delete mailing list and subscribe/unsubscribe members to these lists.
+
+You only need to set your provider's credentials in the config file and start to manage your mailing list without configure http calls, ensure urls or dealing with tricky configurations. Use your time in the business logic of your app.
+
+With Papelillo you can separate in other layer the management of the mailing, so if you decide change the provider or use the implementation for multiples app you don't need to change your core code.
+
+
+## Supported Providers
+
+* Mailgun: Papelillo.Providers.Mailgun
```

Comment:
maybe would be good to add a list of all the possible providers here with a checkbox, so that it helps to keep track about what's implemented and what would be eventually implemented, also for other people using this library, they would see immediately if their provider is supported or not, something like
```
[x] Mailgun
[ ] Gmail
[ ] Mailchimp
```